src/bot_gui.py

```diff
@@ -87,24 +98,24 @@ def display_username_field(self):
         username_label = ttk.Label(self.root, text="Brand Instagram Username:")
         username_label.grid(row=1, column=0, padx=15, pady=5)
         self.username_entry = ttk.Entry(self.root, width=20)
-        self.username_entry.insert(0, "")
+        self.username_entry.insert(0, self.username)
         self.username_entry.grid(row=1, column=1, columnspan=2, padx=15, pady=5)
 
     def display_password_field(self):
         password_label = ttk.Label(self.root, text="Brand Instagram Password:")
         password_label.grid(row=2, column=0)
         self.password_entry = ttk.Entry(self.root, width=20, show="*")
-        self.password_entry.insert(0, "")
+        self.password_entry.insert(0, self.password)
```

Comment:
When we make password `None` by default we can do this, and then it will display a placeholder if there is no password

```suggestion
        if self.password:
             self.password_entry.insert(0, self.password)
```

Reply:
Will do also for username